transcribe.py

```diff
@@ -131,9 +132,12 @@ def main(args, ext_json = ['.json', '.json.gz']):
 		try:
 			tic = time.time()
 			y, ylen = y.to(args.device), ylen.to(args.device)
+			print('Input:', audio_name)
+			print('Xlen:', x.shape[-1] / args.sample_rate)
```

Comment:
what units does `xlen / args.sample_rate` have? Seconds? spectrogram output? Does `--frontend-in-model` option affect this? Please figure this out and add appropriate units to the print message

Reply:
done